apis/nucleus/src/components/listbox/assets/identify.js

```diff
@@ -0,0 +1,9 @@
+export default function identify({ qId, options }) {
+  return {
+    // External or internal ("on the fly") session model.
+    hasExternalSessionModel: !!(qId || options.sessionModel),
```

Comment:
I think it's a little bit confusing/misleading calling it `hasExternalSessionModel`, since it's not an external session model if a qId is specified.

Maybe it's better to call it `isOnTheFlyModel`?

Reply:
Both are a bit confusing, what does on the fly model mean?

Reply:
It's the opposite of an `onTheGroundModel` 😄

Reply:
This is what I'm thinking of when I say "on the fly" in this case: https://qlik.dev/libraries-and-tools/nebulajs/rendering#render-on-the-fly

That is the opposite of an existing object you could say...

But there might be some better naming for it.

Reply:
Why not just hasExistingSessionModel – i.e. `existing` instead of `external`. Would that be more correct?

Reply:
To me, there are two scenarios. When there is an object that already exist in engine, or not.
When passing in either a `qId` or an existing model (`options.sessionModel`), the object exist, otherwise it should be created/generated.

I don't think we need to include sessionModel in the naming, the relevant thing is to indicate if it's an existing object or an object should be created/generated. Therefore, I would rather call it just `existingObject` or something like that.

Reply:
Sounds good 👍